lib/cli.js

```diff
@@ -32,7 +35,7 @@ if (process.argv.join('').indexOf('mocha') === -1) {
     if (!validateMessage(msg)) {
       var incorrectLogFile = (
         isFile
-        ? commitMsgFileOrText.replace('COMMIT_EDITMSG', 'logs/incorrect-commit-msgs')
+        ? commitMsgFileOrText.replace(filename, 'logs/incorrect-commit-msgs')
```

Comment:
It looks to me like `filename` will *always* be `GITGUI_EDITMSG` now and the original `COMMIT_EDITMSG` will never be checked. Why is the current functionality being removed here?